lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%# frozen_string_literal: true %>
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
+  <%%= f.error_notification %>
+  <%%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
```

Comment:
- [ ] use a helper method to render this line if the condition is met call the helper method here instead of having the if logic straight in this view